### ISSUE TITLE
kvm_set_phys_mem: error registering slot: Inappropriate ioctl for device

### ISSUE THREAD

**Issue by jiayizhen94** [Cuju-ft/Cuju#78]
I tried to install Cuju in a vmware virtual machine. This error occurred when I was running the QEMU image. Is there a solution?
`
W: /etc/qemu-ifup: no bridge for guest interface found
kvm_set_phys_mem: error registering slot: Inappropriate ioctl for device
Aborted (core dumped)
`

**Comment by PJTPJT** [comment]
Do you use our kvm module? in folder Cuju/kvm

**Comment by jiayizhen94** [comment]
Yes, I can compile it and run the ./reinsmodkvm.sh script correctly.
But the problem mentioned above will happen when running qemu image.

**Comment by PJTPJT** [comment]
Could you please post your script that start the VM?
And your execution environment, thank you

**Comment by PJTPJT** [comment]
> Yes, I can compile it and run the ./reinsmodkvm.sh script correctly.
> But the problem mentioned above will happen when running qemu image.

Hello? Any update about this?
I think the cause of this problem should be a mismatch between qemu binary and kvm module
Please make sure you have replaced these two parts with the Cuju version. Thank you
